tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/util/K8sUtil.java

```diff
@@ -222,4 +225,15 @@ public static void waitForHiveMQPlatformStateRunning(
         hivemqCustomResource.waitUntilCondition(getHiveMQPlatformStatus("RUNNING"), 5, TimeUnit.MINUTES);
         assertThat(hivemqCustomResource.get().get("status").toString()).contains("RUNNING");
     }
+
+    /**
+     * Waits for all the pods to be deleted in the given namespace
+     * @param client    the Kubernetes client to use
+     * @param namespace the namespace to wait to check for all the pods to be removed
+     */
+    public static void waitForNoPodsDeletedInNamespace(
```

Comment:
```suggestion
    public static void waitForAllPodsDeletedInNamespace(
```

Reply:
thanks! didn't realize what a weird name I used for that! :)